src/util.cc

```diff
@@ -835,6 +837,94 @@ double GetLoadAverage() {
 }
 #endif // _WIN32
 
+double GetCPUWaitRatio(size_t subproc_number, int parallelism) {
```

Comment:
According to Google C++ style  guide, it should rather be called `GetCpuWaitRatio(...)`

Quoting https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

> When an abbreviation is used, prefer to capitalize it as a single "word", e.g., StartRpc() rather than StartRPC()